.github/workflows/release.yml

```diff
@@ -117,6 +117,7 @@ jobs:
           DOWNLOAD: ${{ needs.source.outputs.download }}
           CHECKSUM: ${{ needs.source.outputs.checksum }}
           TAG_BODY: ${{ needs.source.outputs.body }}
+          GH_TOKEN: ${{ secrets.COCKPITUOUS_TOKEN }}
```

Comment:
Please remove the `GH_TOKEN` from line 97, or update it to COCKPITUOUS_TOKEN. I don't think anything else here actually uses a token, so keeping it local to this step seems fine.